dm/syncer/status.go

```diff
@@ -30,9 +30,9 @@ import (
 func (s *Syncer) Status(sourceStatus *binlog.SourceStatus) interface{} {
 	syncerLocation := s.checkpoint.FlushedGlobalPoint()
 	st := &pb.SyncStatus{
-		TotalEvents:         s.count.Load(),
-		TotalTps:            s.totalTps.Load(),
-		RecentTps:           s.tps.Load(),
+		TotalRows:           s.count.Load(),
```

Comment:
should not break compatibility

Reply:
kept it https://github.com/pingcap/tiflow/pull/7511/commits/76c13bab77bf040ff61ca75aee931f2bfe16d8b9